src/Microsoft.Health.Fhir.Shared.Core/Features/Search/SearchOptionsFactory.cs

```diff
@@ -170,6 +170,12 @@ public SearchOptions Create(string compartmentType, string compartmentId, string
 
             searchOptions.UnsupportedSearchParams = unsupportedSearchParameters;
 
+            // Sort is currently not implimented. The sort parameters are added to the search options to allow for future development.
+            if (searchParams.Sort != null && searchParams.Sort.Count > 0)
+            {
+                searchOptions.Sort = searchParams.Sort.Select(sorting => Tuple.Create(sorting.Item1, (SortOrder)sorting.Item2));
```

Comment:
let's do Enum.Parse instead of direct casting. In case the value changes (which is unlikely... but you never know), we would fail as oppose to cast it to something unexpectedly.